components/sidebar/sidebar.jsx

```diff
@@ -583,6 +589,24 @@ class Sidebar extends React.PureComponent {
                     id='sidebarChannelContainer'
                     className='nav-pills__container'
                 >
+                    {pluginApps && pluginApps.length > 0 &&
```

Comment:
Can we add some snapshot testing to the sidebar testing the case with no pluginApps and with some plugins apps?